docs/FAQ.md

```diff
@@ -100,13 +100,15 @@ Alternatively, you can use [jj-fzf](https://github.com/tim-janik/jj-fzf), where
 
 The wiki lists additional TUIs and GUIs beyond the terminal: [GUI-and-TUI](https://github.com/jj-vcs/jj/wiki/GUI-and-TUI)
 
-### How do I keep `jj squash` non-interactive for scripts or agents?
+### How do I keep `jj squash`/`jj split` non-interactive for scripts or agents?
 
 Pass `--no-editor` to accept the default combined description when `jj squash` would otherwise open
 an editor (for example, when both the source and destination commits have descriptions). You can
 also keep the editor closed by providing a description directly with `--message` or by reusing the
 destination description with `--use-destination-message`.
 
+Use `--no-editor` with `jj split` to avoid description prompts; the selected commit keeps its
+current description and the other split commit is left empty.
```

Comment:
nit: missing newline before next paragraph

Reply:
fixed in https://github.com/jj-vcs/jj/compare/9fd091656b3bb7c478e98993e7be221f38407f20..766860c438688b711b6193b31a4e19276a11c190